### PR TITLE
Change version for helm charts in README

### PR DESCRIPTION
**Pull request by gawsoftpl**
Update versions for charts in README files.

### CODE REVIEW

File: charts/kserve-resources/README.md [hunk 1]
```diff
@@ -5,11 +5,11 @@
 First install the `kserve-crd` chart
 
 ```console
-helm install kserve-crd oci://ghcr.io/kserve/charts/kserve-crd --version 0.11.2
+helm install kserve-crd oci://ghcr.io/kserve/charts/kserve-crd --version v0.12.0-rc0
```
Comment (yuzisun) [inline charts/kserve-resources/README.md:8]: This would change every time we release, does it default to the latest version ?

**Review by terrytangyuan — COMMENTED** [review]
We should automatically bump this version as well as part of https://github.com/kserve/kserve/issues/3342

Also there is a tool to automatically generate docs for helm charts.

**Comment by yuzisun** [comment]
> We should automatically bump this version as well as part of #3342
> 
> Also there is a tool to automatically generate docs for helm charts.

We would need to generate the doc if we want to make helm as official installation method

**Comment by terrytangyuan** [comment]
This is fine to fix the current documentation. I created https://github.com/kserve/kserve/issues/3356 to track the docs generation separately.

/lgtm

**Pull request merged**